Conche/Source.swift

```diff
@@ -7,6 +7,22 @@ public protocol SourceType {
   func update() throws
 }
 
+public class LocalFilesystemSource : SourceType {
+  let path:Path
+
+  public init(path:String) {
```

Comment:
I think we should pass in a `Path` instead of string. Especially considering the usage is:

```swift
LocalFilesystemSource(path: Path.current.description)
```